src/main/java/com/checkmarx/flow/CxFlowRunner.java

```diff
@@ -58,7 +58,7 @@
     private final ConfigurationOverrider configOverrider;
     private final List<VulnerabilityScanner> scanners;
     private final ThresholdValidator thresholdValidator;
-    private static final String ERROR_BREAK_MSG = "Exiting with Error code 10 due to issues present";
+    private static final String ERROR_BREAK_MSG = "Exiting with Error code 10 due to checkmarx findings";
```

Comment:
If we are already here, let's format this message using ExitCode.BUILD_INTERRUPTED.getValue(). Besides, the checkIfBreakBuild method already contains some logging. We could combine it with ERROR_BREAK_MSG somehow to avoid duplicate logging.

Also, "Checkmarx" should be capitalized.

Reply:
Fixed